src/headers/tomcrypt_pk.h

```diff
@@ -529,6 +529,10 @@ typedef struct ltc_asn1_list_ {
    unsigned long size;
    /** The used flag, this is used by the CHOICE ASN.1 type to indicate which choice was made */
    int           used;
+   /** Flag used to indicate optional items in ASN.1 sequences */
+   int           optional;
```

Comment:
if we go for possibility 2 regarding the `tag`: How about starting to use `<stdbool.h>`? as this is clearly a boolean :-)